### DISCUSSION TITLE
Is there a way to change the request body in try-it with the chosen oneOf request body

### DISCUSSION THREAD

**Discussion by yedu-alokin** [stoplightio/elements#2554] [Q&A]
Hi, I have noticed that the request body in the Try It feature will not be changed when we have `oneOf` in the `requestBody`. Is there a way to render the request body the same as the chosen option of `oneOf` in the `requestBody`?

**Comment by kentbulza** [comment]
You can load multiple `examples`, each demonstrating a variety of permitted request body to the endpoint. These appear in a dropdown, each displaying as a theoretical request body. Pseudo-code snippet that would appear as your paths.*pathname*.requestBody.content.application/json.examples:
``

    "examples": {
        "This appears in a dropdown": {
            "value": {
                "a": "b"
            },
            "summmary": "This summary does not display.",
            "description": "This description does not display."
        },
        "This also appears in a dropdown": {
            "value": {
                "a": "b",
                "b": 1
            },
            "summmary": "This summary also does not display.",
            "description": "This description also does not display."
        }
    }
``
Note: `examples` is not an array. It's an object of named objects.
